Will not run with JDK 22 under Windows 11

I installed JDK 22 because JDK 20 is not available.  
PS E:\devdrive\rctab\rcv> java -version
openjdk version "22" 2024-03-19
OpenJDK Runtime Environment (build 22+36-2370)
OpenJDK 64-Bit Server VM (build 22+36-2370, mixed mode, sharing)

I cloned the respository and typed ./gradlew run in the rcv folder as instructed. 

PS E:\devdrive\rctab\rcv> ./gradlew run
Downloading https://services.gradle.org/distributions/gradle-8.1.1-bin.zip
...........10%............20%............30%............40%............50%............60%............70%............80%...........90%............100%

Welcome to Gradle 8.1.1!

Here are the highlights of this release:
 - Stable configuration cache
 - Experimental Kotlin DSL assignment syntax
 - Building with Java 20

For more details see https://docs.gradle.org/8.1.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file 'E:\DevDrive\RCTab\rcv\settings.gradle' (C:\Users\david\.gradle\caches\8.1.1\scripts\8mhh30znl8mydwkpfn0xxm1nb).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 66

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 14s

That's it.  I realize this may not actually be a problem with the project. 

AI says: The error message "Unsupported class file major version 66" suggests that there is a mismatch between the Java version used to compile the Gradle script and the Java version being used to run Gradle.

However, since it says JDK 20 or later, and JDK 20 itself isn't availble, I'll wait for clarification.

I tried again with JDK 21.  Very similar error: 

PS E:\devdrive\rctab\rcv> ./gradlew run
Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details

FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file 'E:\DevDrive\RCTab\rcv\settings.gradle' (C:\Users\david\.gradle\caches\8.1.1\scripts\8mhh30znl8mydwkpfn0xxm1nb).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
PS E:\devdrive\rctab\rcv>

--- a/README.md
+++ b/README.md
@@ -32,7 +32,7 @@ On Linux, you may install the .deb file, then run `/opt/rcv/bin/RCTab` to launch
 
 #### Method 2 (Less Easy): Compile and Run Using Gradle
 
-1. Install [JDK 20 or higher](https://jdk.java.net/), and make sure your Java path is picking it up properly by
+1. Install [JDK 20](https://www.oracle.com/java/technologies/javase/jdk20-archive-downloads.html), and make sure your Java path is picking it up properly by
    verifying that the following command returns the expected version:
     
     `$ java -version`

